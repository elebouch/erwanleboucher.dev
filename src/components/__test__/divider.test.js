import React from "react"
import { render } from "@testing-library/react"
import "jest-styled-components"

import Divider from "../divider"

describe("Divider", () => {
  it("renders correctly", () => {
    const { container } = render(<Divider />)
    expect(container.innerHTML).toMatchSnapshot()
  })
})
