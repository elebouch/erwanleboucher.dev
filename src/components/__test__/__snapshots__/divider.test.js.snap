// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider renders correctly 1`] = `
<div
  className="sc-AxjAm gUIqk"
/>
`;
